recipes/opencl-clhpp-headers/all/conandata.yml

```diff
@@ -0,0 +1,13 @@
+sources:
+  # "2023.04.17":
+  #   url: "https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2023.04.17.tar.gz"
+  #   sha256: "179243843c620ef6f78b52937aaaa0a742c6ff415f9aaefe3c20225ee283b357"
+  # "2023.02.06":
+  #   url: "https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2023.02.06.tar.gz"
+  #   sha256: "2726106df611fb5cb65503a52df27988d80c0b8844c8f0901c6092ab43701e8c"
```

Comment:
Why does is not possible to include those newer versions?

Reply:
because those versions depend on C OpenCL headers which were merged recently https://github.com/conan-io/conan-center-index/pull/17417
Uncommented these lines that PR has been merged